docs/design/LAG-for-ports/README.md

```diff
@@ -0,0 +1,50 @@
+# LAG for ports
+
+## Overview
+
+Link aggregation provides ability to combine multiple physical connections into one logical connection to improve resiliency. Link aggregation group (LAG) is a group of ports associated with the logical port on the switch.
+
+## API
+
+* Get existing LAG logical ports on the switch `GET /v2/switches/{switch_id}/lags`. Response example:
+~~~
+[ 
+    {
+        "logical_port_number": 2001,
+        "port_numbers": [1, 2, 3]
+    },
+...
+]
+~~~
+
+* Create LAG logical port on the switch `POST /v2/switches/{switch_id}/lags` with body:
+~~~
+{
+    "port_numbers": [1, 2, 3]
+}
+~~~
+Response example:
+~~~
+{
+    "logical_port_number": 2001,
+    "port_numbers": [1, 2, 3]
+}
+~~~
+
+* Delete LAG logical port on the switch `DELETE /v2/switches/{switch_id}/lags/{logical_port_number}`.
+
+
+## Details
+All logical port related commands are sent to the switches using gRPC speaker.
+
+Open-kilda calculate logical port number using the following rule: 2000 + min of the physical ports number in the LAG. It is not allowed to have one physical port in two LAGs so this rule will provide unique logical port number for any port configuration.
```

Comment:
I'm not sure about unique port numbers. 
exaple:
[1, 2] ports will have logical port number 2001
[2, 3] ports will have logical port number 2002

logical ports are unique but both ports use port 2.

Reply:
The same physical port can't be a part of two LAG ports. So this configuration is not valid. Added a validation step to the description.